src/components/select/select.scss

```diff
@@ -48,7 +47,8 @@
         }
 
         .mdc-floating-label {
-            top: $floating-label-top-value;
+            @include looks-like-input-label;
+            top: 50%;
```

Comment:
I'm not sure, should we have the label _not_ float when the selected item's text is empty?

![image](https://user-images.githubusercontent.com/154725/102219435-c34a7d00-3edf-11eb-906f-1b9b2862dd52.png)

It looks a little odd with the floating label, but on the other hand, it does make it clear that "nothing is selected"…

Regardless, I don't think we _have_ to change it in this PR, right?

Reply:
It does, but it's because they have set an empty choice. I think it's their own hack, and if you open the list, the first item is empty. 

I don't know why it's like that. But I don't think we have to do anything about it here.

Reply:
That's the way Lime CRM handles selecting "nothing" in a list of options: there has to be an option with no description, representing the option to select "nothing"…

Reply:
The difference is that limel-select expects a "nothing"-option to have an empty value for both `value` and `text`, while Lime CRM requires the "nothing"-option to have a `value`.

But yeah, let's just ignore that for now. We can take care of it later.